### PR TITLE
Menu reordering

### PR DESCRIPTION
**Pull request by lguima**
It's related to the issue #3277

Changes in this pull request:

- Reordered and grouped some items in the sidebar menu according to their frequency of use and to encourage the use of them

@JC5

### CODE REVIEW

**Comment by lguima** [comment]
@JC5 
To create a string translatable, should I place it in all language files or only in the en-US?

**Comment by JC5** [comment]
Putting the strings in en_US is enough.

File: resources/views/v1/layout/default.twig [hunk 1]
```diff
@@ -86,9 +86,27 @@
...
-                        </span>
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                                <span class="visible-xs">@</span>
+                                <span class="hidden-xs">{{ Auth.user.email }}</span>
+                            </a>
+
+                            <ul class="dropdown-menu" style="width: unset;">
+                                <li class="user-footer">
+                                    <div class="col-xs-6 text-center">
+                                        <a href="{{ route('profile.index') }}" class="btn btn-default btn-flat">
```
Comment (JC5) [inline resources/views/v1/layout/default.twig:97]: I'm not a fan of buttons here. They render badly and it's unexpected UI. Can this be a dropdown list?

**Comment by JC5** [comment]
Dropping the icons for the submenu's has the weird effect that the sub-items are more to the left than the main items. I suggest you leave the icon for the sub-items.

Budget icon is OK. Use the empty calendar version for bills. Don't use the hamburger menu icon for options. Maybe [sliders](https://fontawesome.com/v4.7.0/icon/sliders)?

File: resources/views/v1/layout/default.twig (hunk 1 above)
Comment (lguima) [reply resources/views/v1/layout/default.twig:97]: Actually, this is from [the official main header template](https://adminlte.io/docs/2.4/main-header), what is painful to work without the original layout.

Also it broken when I was taking screenshots and replace my email address to a smaller one.

Then, I'm reverting, I'll left in the sidebar menu.

**Comment by lguima** [comment]
> Budget icon is OK. Use the empty calendar version for bills. Don't use the hamburger menu icon for options. Maybe [sliders](https://fontawesome.com/v4.7.0/icon/sliders)?

Agreed, I updated the icons.

> Dropping the icons for the submenu's has the weird effect that the sub-items are more to the left than the main items. I suggest you leave the icon for the sub-items.

Indeed, but the Font Awesome 4.7 is very limited, I also used some of the previous sub-menu icons for some menu items, then it will cause duplications.

But looking for the official live preview they use a [circle-o](https://fontawesome.com/v4.7.0/icon/circle-o) for each sub-menu item, but it doesn't seems good to me.

Then I tried with [angle-right](https://fontawesome.com/v4.7.0/icon/angle-right) and it seems better.

sidebar-menu

**Comment by JC5** [comment]
Yeah that could work too.

**Comment by lguima** [comment]
Great, thanks! I marked the PR as **Ready for Review** now.

**Comment by JC5** [comment]
:+1:

**Pull request merged**